quickstart/201-azfw-with-secure-hub/variables.tf

```diff
@@ -0,0 +1,30 @@
+// Create Variables for Location and Tags
+variable "location" {
+  default = "eastus"
+}
+variable "tags" {
```

Comment:
Unless the tags are required to make this sample work, I recommend we remove them.

Reply:
removed tags